manage.py

```diff
@@ -24,7 +24,7 @@
 
 from django.core.management import execute_from_command_line
 
-from pootle import syspath_override
+from pootle import checks, syspath_override
```

Comment:
So you were complaining this is nasty... From what I see the cleanest (and suggested) way to register these checks is using an [`AppConfig`](https://docs.djangoproject.com/en/1.7/ref/applications/#django.apps.AppConfig), more specifically its [`ready()` method](https://docs.djangoproject.com/en/1.7/ref/applications/#django.apps.AppConfig.ready). Since 1.7 this is the place to run initialization code like this.

To learn more about this, [see the example from django.contrib.auth](https://github.com/django/django/blob/0ed7d155635da9f79d4dd67e4889087d3673c6da/django/contrib/auth/apps.py).